### PR TITLE
fix(import): Show the error message with db install guide when exist the importing db issue

### PR DESCRIPTION
**Pull request by prosdev0107**
### SUMMARY
Add error message when user tries to import without having db driver installed

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE:
Clipboard 2022-23-06 at 11 02 20 PM
Clipboard 2022-23-06 at 11 02 45 PM

AFTER:

https://user-images.githubusercontent.com/47900232/176767158-36c312bd-b20a-4500-ba68-b30f51bd26d3.mov

### TESTING INSTRUCTIONS
1. Go to Connect Database Modal
2. Select the import database from invalid file.

[In case of Invaid db file with already installed driver](https://github.com/apache/superset/files/9023038/database_export_20220628T230202.2.zip)
[In case of invaoide db fiel with not installed driver](https://github.com/apache/superset/files/9023039/database_export_20220628T230242.zip)

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/ImportModal/ErrorAlert.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ const ErrorAlert: FunctionComponent<IProps> = ({
           {t(
             'Database driver for importing maybe not installed. Visit the Superset documentation page for installation instructions:',
           )}
+          &nbsp;&nbsp;
```
Comment (Antonio-RiveroMartnez) [inline superset-frontend/src/components/ImportModal/ErrorAlert.tsx:54]: Wondering why not using css on the anchor or span to space out?

**Comment by yousoph** [comment]
A few text updates needed: 
When the error shows up on the password screen: 
image
Period missing before "Please" and there's an extra space between the : and here 

When the error shows up on the db connection screen, a period is also missing before "Please" 
image

**Comment by prosdev0107** [comment]
@yousoph 
Resolved your review. Please kindly check this.

**Review by Antonio-RiveroMartnez — APPROVED** [review]
LGTM!

**Review by rusackas — APPROVED** [review]

**Pull request merged**